### DISCUSSION TITLE
[Docs] 自动化文档生成工具

### DISCUSSION THREAD

**Discussion by zxq08** [element-plus/element-plus#8680] [Feature Request]
Feature type: **`Docs`**

## Details

- Enhanced component name: `All`

## Description

请问目前el组件库是如何快速生成api表格的？
手动维护或者用到了什么插件自动生成吗？

### Additional comments

_(empty)_

**Comment by KimYangOfCat** [comment]
看了源码，目前应该是手动维护的

**Reply by zxq08** [reply]
谢谢，我担心他们是有单独的库打包进来的
